sdk/eventgrid/microsoft-azure-eventgrid/src/main/java/com/microsoft/azure/eventgrid/models/MediaJobErrorCategory.java

```diff
@@ -28,7 +28,10 @@
     CONFIGURATION("Configuration"),
 
     /** The error is related to data in the input files. */
-    CONTENT("Content");
+    CONTENT("Content"),
+
+    /** The error is related to account information. */
+    ACCOUNT("Account");
```

Comment:
This is track 1 code. Why are we updating track 1? Are we planning to release Track 1 too?